packages/components/addon/components/hds/dropdown/toggle-user.hbs

```diff
@@ -1,11 +1,11 @@
 <button
   class={{this.classNames}}
-  ...attributes
   aria-label={{@text}}
   type="button"
   disabled={{if this.isDisabled "disabled" null}}
+  ...attributes
 >
-  <div class="hds-dropdown-toggle__user">
+  <div class="hds-dropdown-toggle-user--avatar">
```

Comment:
[chore] this is not a _modifier_, is an [_element_](http://getbem.com/naming/), so should be `hds-dropdown-toggle-user__avatar`

Reply:
it's a "type" of toggle/user...but tbh I'm getting a little mixed up here so I'm drawing up a visualization for us to review.